src/twig/linearize/test/histogram.test.cpp

```diff
@@ -14,13 +14,13 @@ SCENARIO("linearizing a histogram region"){
 
   auto first = fine.begin();
   linearize::histogram( first, fine.end() - 1,  histogram.begin(), sink );
-  REQUIRE( first == (fine.end() - 1) );
+  CHECK( first == (fine.end() - 1) );
```

Comment:
As above: this is end() - 1, thus the LAST element, not the end of the sequence.